scheduler/scheduler.go

```diff
@@ -45,6 +45,10 @@ const (
 var (
 	authProvider = flag.String("mesos_authentication_provider", sasl.ProviderName,
 		fmt.Sprintf("Authentication provider to use, default is SASL that supports mechanisms: %+v", mech.ListSupported()))
+	schedulerHost = flag.String("mesos_scheduler_ip", "0.0.0.0",
+		fmt.Sprintf("TCP4 Address for the Scheduler to bind"))
```

Comment:
you can run go vet from the terminal, it checks for things that "could" be errors, but it is not correct 100% of the time. More info here https://golang.org/cmd/go/#hdr-Run_go_tool_vet_on_packages
I actually didn't run go vet on this file but the error you get is because `Sprintf` expects a format string and then the value(s) to use, in this case you are not passing any variables to use, so you can simply use `fmt.Println()`